LiskHQ/lisk-sdk
# Add bft header - Closes #4247

### What was the problem?
BFT header was not added during the creation of block

### How did I solve it?
Add the header while creating the block

### How to manually test it?
Start the node with devnet setting, and observe block header changes after 101 blocks

### Review checklist

- [ ] The PR resolves #4247
